package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
+    "test": "karma start karma.conf.js",
```

Comment:
So, since we are testing both client and server, change this to `test:client`, and the previous one that should be using grunt to `test:server`. So we have two testing npm scripts